src/app/window-handler.ts

```diff
@@ -1006,6 +1011,17 @@ export class WindowHandler {
         reloadWindow(focusedWindow as ICustomBrowserWindow);
     }
 
+    /**
+     * Zoom in
+     */
+    private onZoomIn(): void {
```

Comment:
@johankwarnmarksymphony Is this a workaround? do we have an open issue in electron? if so, can you please link to this PR? Thanks

Reply:
Yes, we are already using the accelerator
zoomIn: 'Ctrl+=',
This accelerator works on windows but not on mac.
So on mac we need to listen for Command/= too